### ISSUE TITLE
[SF][ui5-table] Add "row navigated" feature 

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents#5044]
### **Feature Request Description**
The SAPUI5 Responsive Table (sap.m.Table) allows to display a navigation indicator at the end of the row:
![Screenshot 2022-04-11 at 18 27 54](https://user-images.githubusercontent.com/7316793/162786954-750f91c0-8f35-4e7c-af48-3899530db694.png)

See the[ Fiori Guidelines](https://experience.sap.com/fiori-design-web/responsive-table/#indicate-navigated-item) and the [SAPUI5 code sample](https://sapui5.hana.ondemand.com/#/entity/sap.m.Table/sample/sap.m.sample.TableNavigated).
It would be nice to have a similar feature in UI5-WebComponents Responsive Table (`ui5-table`).

### **Proposed Solution**
Add a `navigated` boolean attribute to the `ui5-table-row` component which has the same behavior as SAPUI5 Responsive Table

```
<ui5-table-row navigated>
	<ui5-table-cell>Notebook Basic 15HT-1000</ui5-table-cell>
	<ui5-table-cell>Very Best Screens</ui5-table-cell>
	<ui5-table-cell>30 x 18 x 3cm</ui5-table-cell>
	<ui5-table-cell>4.2KG</ui5-table-cell>
	<ui5-table-cell>956EUR</ui5-table-cell>
</ui5-table-row>
```

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: SF

**Comment by stbodurov** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

Could you please evaluate if this feature request is feasible?

Regards,
Stanislav

**Comment by Neeeko** [comment]
Hi team! Any updates on this ticket? It's quite a huge feature for us as our application relies a lot on a navigation workflow using the table. Thanks!

**Comment by nikolay-kolarov** [comment]
Hi @Neeeko,

We have no update so far. Thanks for reaching out and providing this additional info.
We will prioritize the item accordingly and try to include it in one of the upcoming waves.

Best Regards,
Nikolay

**Comment by jatinthaker** [comment]
Can we get an update on when this is being planned on being delivered into WC-UI5? Discussed with Stoehr. Ok with Medium Priority.

**Comment by nikolay-kolarov** [comment]
Hi @jatinthaker ,

We plan to include it in the next sprint planning and if there are no unexpected obstacles you can expect the functionality in the end of September.
I will update you, once we have progress.

Best Regards,
Nikolay

**Comment by ivoplashkov** [comment]
Hello,

The pull request has been merged and the "navigated" property can be consumed, however the property is currently not working in Safari, therefore we will keep the issue opened until we fix it there as well.

Best regards,
Ivaylo

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rl  1.9.0 has been released on Monday 31.10, so I am moving the issue to 1.10.0 milestone ( to be able to close the 1.9.0 milestone as it expired).

**Comment by jatinthaker** [comment]
ok. are we looking to close this in 1.10.0? Looks like except Safari the other browsers are covered.

**Comment by ivoplashkov** [comment]
Hello @jatinthaker,

The feature is working in 1.9.0 on all browsers, we have done some small adjustments to the accessibility that will be consumed with 1.10.0, therefore I am closing this issue as completed.

Best regards,
Ivaylo

**Comment by jatinthaker** [comment]
Excellent and thanks for the update.

Thanks

Jatin

"Arise Awake Stop not till the goal is reached"
-- Swami Vivekananda

On Nov 3, 2022, at 5:07 AM, Ivaylo Plashkov ***@***.***> wrote:

﻿

Closed #5044<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSAP%2Fui5-webcomponents%2Fissues%2F5044&data=05%7C01%7Cjatin.thaker%40sap.com%7C2795a3a0d48049c64f8708dabd93ff17%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638030740610343906%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=1jVSlhwdZ2kDyeCkKwroMLco4xKpuI4ay%2FB5dZOL7%2FQ%3D&reserved=0> as completed.
